<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViWindow</class>
 <widget class="QWidget" name="ViWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>319</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ViWindowControlWidget" name="topLeft" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="ViWindowControlWidget" name="topRight" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="ViWindowControlWidget" name="bottomLeft" native="true"/>
   </item>
   <item row="3" column="3">
    <widget class="ViWindowControlWidget" name="bottomRight" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="ViWindowControlWidget" name="bottom" native="true"/>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="ViWindowControlWidget" name="left" native="true"/>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="ViWindowControlWidget" name="right" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="ViWindowControlWidget" name="titleBar" native="true">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>247</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QWidget" name="content" native="true">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ViWindowControlWidget" name="top" native="true"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QWidget" name="controlWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="ViButton" name="minimizeButton" native="true"/>
      </item>
      <item>
       <widget class="ViButton" name="maximizeButton" native="true"/>
      </item>
      <item>
       <widget class="ViButton" name="fullscreenButton" native="true"/>
      </item>
      <item>
       <widget class="ViButton" name="closeButton" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViWindowControlWidget</class>
   <extends>QWidget</extends>
   <header>viwindowcontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViButton</class>
   <extends>QWidget</extends>
   <header>vibutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
